rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+  end
```

Comment:
- [ Required ] I think it will be better if the rental adds itself to the `book` variable thus achieving the _Create the belongs-to side of Rental and Book (a rental belongs to a book)._ requirement specified in the project requirements.

- [ Required] I think it would be better if the rental adds itself to the `person` variable thus achieving the _Create the belongs-to side of Rental and Person (a rental belongs to a person)._  specified in the project requirements. Please see the screenshot below and rectify this.

## Example code
```ruby
class Rental
  attr_accessor :date, :person, :book

  def initialize(date, book, person)
    @date = date

    @person = person
    person.rentals << self

    @book = book
    book.rentals << self
  end
end
```
## Project requirements
![Screenshot from 2023-05-25 01-10-32](https://github.com/BahirHakimy/OOP-school-library/assets/16392046/2a69f107-797d-4070-8ba2-74835cc4c23e)